docs/dev-guide/guide.md

```diff
@@ -254,7 +301,7 @@ const question = await queryRow(sql.select_question, { question_id: 45 }, Questi
 
 ## Database schema modification
 
-See [`migrations/README.md`](https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/migrations/README.md)
+See the [migrations page](./migrations/README.md).
```

Comment:
I'm really really not sure about this. In isolation, there's no way to know what file this points to without understanding how the docs are build.

_Do_ we just have to know that, and rely on always reading the dev guide as published in the docs?

Reply:
I always read the dev guide as published in the docs, and I found the click away a little annoying, especially from a discoverability perspective.

Reply:
I would be curious if anyone reads the dev guide outside of the docs.

Reply:
To be fair, I only really do when I'm editing it. I guess this is optimizing for consumption rather than authoring?

```suggestion
See the [migrations documentation](./migrations/README.md).
```

Reply:
Yep!